### PR TITLE
[WIP] add check for completed claim, function to sync errored rows

### PR DESCRIPTION
**Pull request by tdquan**

### CODE REVIEW

File: api/routes/webhooks.py [hunk 1]
```diff
@@ -27,3 +30,20 @@ def create_entity_from_row(entity_name):
             "exception": "couldn't complete your request",
             "details": e
         }), 500
+
+
+@app.route('/webhooks/sync_airtable', methods=['POST'])
+def sync_airtable():
+    '''
+    This is to be converted to a celery task once the pipeline is completed as to prevent timed-out response
```
Comment (Ledoux) [inline api/routes/webhooks.py:38]: so ...I don't think we would need that now based on the new after insert pipelines. :p sorry...

Comment (Ledoux) [reply api/routes/webhooks.py:38]: or may I be wrong ? perhaps

Comment (tdquan) [reply api/routes/webhooks.py:38]: Yes I think if you have a task that will sync the airtable then this route can be refactored to use that instead. The idea is to have a button on the airtable which Manu can press to call this route and force-sync the whole base when necessary.

**Comment by Ledoux** [comment]
ok, that 's a good point ! 
So this route should now call the `tasks.science_feedback.sync_with_rows(mode='all')`
Either with .delay(), or not...I.e. we call it synchronously, I don't know what is the best for airtable webhook.

**Comment by Ledoux** [comment]
@tdquan I thinks this pull request is not anymore meaningful ? I will close it. Thanks

**Pull request closed without merging**